src/App.vue

```diff
@@ -94,6 +94,25 @@
       </div>
     </v-fade-transition>
 
+    <!-- Starting Amalgamation spinner -->
+    <v-fade-transition>
```

Comment:
I'm using a new spinner now that says: "Starting Amalgamation".

I need a spinner since we're making a network call to create the temp business and it just looks so weird not having one. I could've used the "Fetching Data" spinner but the wording is inaccurate so I didn't like it.

Sev, please let me know if you prefer something else.

Reply:
I agree that you should have this spinner, but I wonder if it could be easily merged into the fetching data spinner above?

An idea: instead of a boolean to show/hide the spinner, use a string (which either has a value or doesn't). That same string would be the spinner text.

Reply:
Looks good! Resolving.